prepare-docker.sh

```diff
@@ -26,6 +26,17 @@ export -f stop_docker_service
 # 既存のサービスを落とし、ビルド完了を待つ処理を並列実行する
 echo "stop_docker_service build_image" | xargs -P 0 -n 1 bash -c
 
+if [ $1 == "update-gachadata" ]; then
+  echo Updating gachadata...
+  call docker compose up -d db
+
+  # ここで遅延を入れないとdbが起動する前にgachadataを更新するスクリプトが走ってしまう
+  sleep 3
+  docker exec -it seichiassist-db-1 /update-gachadata.sh
+  echo Completed update gachadata.
```

Comment:
syntax error + quote

```suggestion
  echo "Completed updating gachadata."
```